langchain4j-azure-cosmos-no-sql/src/main/java/dev/langchain4j/store/embedding/azure/cosmos/no/sql/AzureCosmosDBNoSqlEmbeddingStore.java

```diff
@@ -0,0 +1,215 @@
+package dev.langchain4j.store.embedding.azure.cosmos.no.sql;
+import com.azure.cosmos.CosmosClient;
+import com.azure.cosmos.CosmosContainer;
+import com.azure.cosmos.CosmosDatabase;
+import com.azure.cosmos.models.CosmosBulkOperations;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosItemOperation;
+import com.azure.cosmos.models.CosmosQueryRequestOptions;
+import com.azure.cosmos.models.CosmosVectorIndexSpec;
+import com.azure.cosmos.models.CosmosVectorEmbeddingPolicy;
+import com.azure.cosmos.models.PartitionKey;
+import com.azure.cosmos.models.PartitionKeyDefinition;
+import com.azure.cosmos.util.CosmosPagedIterable;
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.store.embedding.EmbeddingMatch;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static dev.langchain4j.internal.Utils.isNullOrBlank;
+import static dev.langchain4j.internal.Utils.isNullOrEmpty;
+import static dev.langchain4j.internal.Utils.randomUUID;
+import static dev.langchain4j.internal.ValidationUtils.ensureTrue;
+import static dev.langchain4j.store.embedding.azure.cosmos.no.sql.MappingUtils.toEmbeddingMatch;
+import static dev.langchain4j.store.embedding.azure.cosmos.no.sql.MappingUtils.toNoSqlDbDocument;
+import static java.util.Collections.singletonList;
+
+public class AzureCosmosDBNoSqlEmbeddingStore  implements EmbeddingStore<TextSegment> {
```

Comment:
```suggestion
public class AzureCosmosDbNoSqlEmbeddingStore  implements EmbeddingStore<TextSegment> {
```
to keep consistent naming

Reply:
Fixed in new iteration.